.github/workflows/ci.yml

```diff
@@ -7,13 +7,18 @@ on:
 
 jobs:
     tests:
-        name: Tests
+        name: "Tests on PHP ${{ matrix.php }}${{ matrix.name_suffix }}"
         runs-on: ubuntu-latest
 
         strategy:
             fail-fast: false
             matrix:
                 php: [ '5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0' ]
+                extensions: ''
```

Comment:
Specifying `extensions` here ensures that the extra job added with a custom value for `extensions` does not swallow the normal job by matching its values.